main/test/src/mill/main/MainModuleTests.scala

```diff
@@ -27,6 +27,12 @@ object MainModuleTests extends TestSuite {
       Map("1" -> "hello", "2" -> "world")
     }
     def helloCommand(x: Int, y: Task[String]) = Task.Command { (x, y(), hello()) }
+    def helloWorker = Task.Worker {
+      // non-serializable, but everything should work fine nonetheless
```

Comment:
What the story behind this comment? We never serialize workers in Mill.

Reply:
The `show` command tries to serialize them nonetheless, with the changes here. The `if t.writerOpt.nonEmpty` in `RunScript.scala` works around that.

Reply:
```suggestion
      // non-json-serializable, but everything should work fine nonetheless
```